main.py

```diff
@@ -0,0 +1,82 @@
+"""
+changelog:
+    3/3/20: Finished implementation of ctrladd, fixed various bugs in actual code (but not pseudocode), changed
+        qubits generation, rewrote main, created changelog and to do.
+    2/26/20: Wrote out ctrladd function in pseudocode, implemented through step 4
```

Comment:
not necessary, because we have git, and the history is stored in the commit messages